src/components/ListDetail/ListDetail.module.css

```diff
@@ -79,17 +79,20 @@
 /**
   * List detail item after
   * 1) When list variant is 'ordered', add a line between items that shouldn't link out unliike the bullet
+  * 2) This sets the size of the gap above the vertical line by determining the top of the line relative to the top of the current list item; a value of 0 would position the line at the top of the current list item; 100% would position it at the bottom of the current list item. We want a small visual gap between the bullet/icon and the top of the line 
+  * 3) This sets the height of the vertical line relative to the height of the current list item. We want a small visual gap between the bottom of the line and the next bullet/icon
+  * 4) Using a border color here because this is a vertical line, similar to a border
   */
 .list-detail__list--ordered .list-detail__item {
   &:after {
     content: '';
     position: absolute;
-    top: 1.5rem;
     left: 0.25rem;
+    top: 1.75rem; /* 2 */
```

Comment:
The gaps above and below the line are subject to change because: a) the icons (success, warning, error) and number are larger than the grey bullet, so when a bullet is present the gap will be slightly larger than when an icon or number is used; b) if the text in a list item takes more than a single line, the gap at the bottom will be larger than the gap at the top.